### ISSUE TITLE
NoBadGoodies

### ISSUE THREAD

**Issue by saamohod** [LoneGazebo/Community-Patch-DLL#9916]
_1. Mod Version (X.Y.Z). Current Version: 3.6.2_

_4. Describe the Issue_
"NoBadGoodies" is written on Extended Information about Scout unit. My guess it means that no angered barbs may spawn upon activating an ancient ruin. But there is no mention of whether it's true or even what NoBadGoodies means on the unit card in Civilopedia.

**Comment by artemigkh** [comment]
You're correct, the only goody hut entries in the db that have the bad flag are the ones that spawn angered barbarians and the function
```
/// Is a Particular Goody ID a valid Goody for a certain plot?
bool CvPlayer::canReceiveGoody(CvPlot* pPlot, GoodyTypes eGoody, CvUnit* pUnit) const
{
...
```
uses this flag to determine whether or not can get the barb spawn event based on this flag

seems like a gap in the documentation, might be nice to add a short description next to that attribute in the UI
